components/atoms/EntitlementBox.js

```diff
@@ -2,7 +2,10 @@ import PropTypes from "prop-types";
 
 export function EntitlementBox(props) {
```

Comment:
EntitlementBox.test.js needs a unit test to make sure it is rendered on Primary state.

Look at DrawerItem.test.js for an example.

All unit tests in EntitlementBox.test.js should be wrapped in a describe() clause.